pattern_import_export/models/ir_exports_line.py

```diff
@@ -137,9 +137,10 @@ def _check_required_fields(self):
     @api.depends("name")
     def _compute_related_level_field(self):
         for export_line in self:
-            if export_line.name and export_line.export_id.model_id.model:
+            # if export_line.name and export_line.export_id.model_id.model:
```

Comment:
You can totally remove this line now, not need to keep some old not working lines even comented